bundles/AdminBundle/Resources/public/js/pimcore/object/tags/hotspotimage.js

```diff
@@ -14,7 +14,7 @@
 pimcore.registerNS("pimcore.object.tags.hotspotimage");
 pimcore.object.tags.hotspotimage = Class.create(pimcore.object.tags.image, {
 
-    type: "hotspotimage",
+    type: "image",
```

Comment:
@mattamon I'm wondering why this `type` has an effect on the asset upload, can you maybe share the details on that? Thanks :)

Reply:
Sure :) It was party of this issue https://github.com/pimcore/pimcore/issues/13000

In this PR https://github.com/pimcore/pimcore/pull/13726 we added the type to `assetSingleUploadDialog` in the helpers.js This was needed to check the type in the backend to make sure e.g. that you cannot upload a PDF to an image tag. 
In the backend we then check for the type https://github.com/pimcore/pimcore/pull/13726/files#diff-8a94426359af1cadbcb7fdeb7af5731ce2ac295a758ed07a38d36db9d5d85ff9R531 and if the type does not match we throw an exception. 

The hotspotimage extends the image so it uses the same functionality and that's why it is failing in the upload dialog.

Reply:
@mattamon ah, ok, this comes from the inherited method from `image.js` 😉 
So we shouldn't change the type then in `hotspotimage.js` but we should just use hardcoded `image` here as the value: 
https://github.com/pimcore/pimcore/blob/4f476328570738270892bda7d785d21617012dbe/bundles/AdminBundle/Resources/public/js/pimcore/object/tags/image.js#L278 

Because this value doesn't have to do anything with the actual data-type, it just specifies the type expected for the upload ...

Reply:
Should we also rename it to `uploadType` then to make it more clear?

Reply:
@mattamon makes sense, yes, maybe even `uploadAssetType` to be even more clear 😊